src/xrpld/ledger/ApplyView.h

```diff
@@ -40,6 +40,9 @@ enum ApplyFlags : std::uint32_t {
 
     // Transaction came from a privileged source
     tapUNLIMITED = 0x400,
+
+    // Transaction is executing as part of a batch
+    tapBATCH = 0x800
```

Comment:
I'm not sure if this matters, but it looks like the next flag should actually be `0x40`